docs/src/tutorials.md

```diff
@@ -1,5 +1,5 @@
 # [Tutorial notebooks](@id tutorials)
-Here are a few tutorials in the form of Jupyter Notebooks to help you get started using Korg.  Each of these is a link to a folder in GitHub containing the notebook (the `.ipynb` file) and any supporting data.  You can download everything if you want to run the notebook yourself, or just view the notebook on GitHub.
+Here are a few tutorials in the form of Jupyter Notebooks to help you get started using Korg.  Each of these is a link to a folder in GitHub containing the notebooks (the `.ipynb` files) in both Julia and Python and any supporting data.  You can download everything if you want to run the notebook yourself, or just view the notebook on GitHub.
```

Comment:
```suggestion
Here are a few tutorials in the form of Jupyter Notebooks to help you get started using Korg.  Each of the following bullets link to a folder on GitHub containing the associated Julia and Python notebooks (the `.ipynb` files) and any supporting data.  You can download everything if you want to run the notebook yourself, or just view the notebook on GitHub.
```

Reply:
I will change this because I messed up: the first two bullets go directly to the notebooks

Reply:
This comment was in error.  I thought this was the repo README.